### PR TITLE
[ADP-1963] publish validity interval data in ApiTransaction from stored cbor

### PR DESCRIPTION
**Pull request by paolino**
I have added validity interval field to `get /transaction` and `get /transactions` by parsing the CBOR from the transaction history

### Issue Number

ADP-1963

### CODE REVIEW

File: lib/wallet/src/Cardano/Wallet/Read/Primitive/Tx/Mary.hs [hunk 1]
```diff
@@ -98,11 +98,11 @@ fromMaryTx tx =
     , anyEraCerts certs
     , assetsToMint
     , assetsToBurn
-    , Just (fromLedgerTxValidity ttl)
+    , Just $ afterShelleyValidityInterval val
     )
   where
     SL.Tx bod wits mad = tx
-    MA.TxBody ins outs certs wdrls fee ttl _upd _adh mint = bod
+    MA.TxBody ins outs certs wdrls fee val _upd _adh mint = bod
```
Comment (HeinrichApfelmus) [inline lib/wallet/src/Cardano/Wallet/Read/Primitive/Tx/Mary.hs]: As the word "validity" is overloaded a lot, I think that `ttl` is a better name than `val`, even though it is slightly wrong. (The validity here is about a time interval in which the transaction is considered applicable to the ledger state.)

Just a nitpick though, happy to merge without this change.

**Review by HeinrichApfelmus — APPROVED** [review]
Looks good and neat to me! 😊 Just a minor nitpick, feel free to merge.

**Comment by paolino** [comment]
bors retry

**Pull request merged**